tools/cargo_package.py

```diff
@@ -49,7 +49,7 @@ def get_version(toml_path):
 def main():
     os.chdir(root_path)
 
-    run(["tools/build.py", "libdeno_static_lib", "--release"])
```

Comment:
1. Add --locked
2. Is there a way to build the required target instead of relying on "it'll probably get built by default" ?

Reply:
1. Done

2. The target is a GN target, so I can't reference it from "cargo build". It is asserted below that it does get built tho.

Reply:
cargo build --manifest-path core/Cargo.toml --lib --release --locked

Reply:
done